src/Application.php

```diff
@@ -325,10 +325,17 @@ public function initProviders()
         // Make sure we keep our current locale..
         $currentlocale = $this['locale'];
 
-        // Setup Swiftmailer, with optional SMTP settings. If no settings are provided in config.yml, mail() is used.
+        // Setup Swiftmailer, with the selected Mail Transport options: smtp or `mail()`.
         $this->register(new Silex\Provider\SwiftmailerServiceProvider());
-        if ($this['config']->get('general/mailoptions')) {
+        if ($this['config']->get('general/mailoptions/transport') == 'mail') {
+            // Use the 'mail' transport.
+            $this['swiftmailer.transport'] = \Swift_MailTransport::newInstance();
```

Comment:
Hi @bobdenotter,

In order to make the 'mail' transport to work, you alos have to set `use_spool` to false. 
See: https://github.com/bolt/bolt/pull/2864#discussion_r25054879